player/video.c

```diff
@@ -119,6 +119,7 @@ void reset_video_state(struct MPContext *mpctx)
     mpctx->mistimed_frames_total = 0;
     mpctx->drop_message_shown = 0;
     mpctx->display_sync_drift_dir = 0;
+    mpctx->display_sync_error = 0.0;
```

Comment:
```suggestion
    mpctx->display_sync_error = 0;
```

Reply:
I originally had it like that but changed it for symmetry with https://github.com/mpv-player/mpv/blob/master/player/loadfile.c#L1601 but I guess following local style makes more sense

Reply:
We generally do `0`, and even few lines about `last_av_difference` is double too.

Reply:
I mean it is a double though. Not like the implicit cast is the end of the world I guess.